src/cmd/build.js

```diff
@@ -12,9 +12,47 @@ import {createLogger} from '../util/logger';
 
 const log = createLogger(__filename);
 
+// Flow Types
+
+import type {OnSourceChangeFn} from '../watcher';
+import type {ExtensionManifest} from '../util/manifest';
+
+export type PackageCreatorFn =
+    (params: PackageCreatorParams) => Promise<ExtensionBuildResult>;
+
+export type ExtensionBuildResult = {
+  extensionPath: string,
+};
+
+export type PackageCreatorParams = {
+  manifestData?: ExtensionManifest,
+  sourceDir: string,
+  fileFilter: FileFilter,
+  artifactsDir: string,
+};
+
+export type BuildCmdParams = {
+  sourceDir: string,
+  artifactsDir: string,
+  asNeeded?: boolean,
+};
+
+export type BuildCmdExtraParams = {
+  manifestData?: ExtensionManifest,
+  fileFilter?: FileFilter,
+  onSourceChange?: OnSourceChangeFn,
+  packageCreator?: PackageCreatorFn,
+};
+
+export type FileFilterOptions = {
```

Comment:
I know you placed all of the exported type definitions in one place but I think it would make the code easier to read if each signature type was directly above the function or class that it pertains to. This is because, in most cases, when you're reading a function, you will need to reference its signature type.